diff --git a/src/SKSE/Logger.cpp b/src/SKSE/Logger.cpp
index d16fedd3..b36fd59f 100644
--- a/src/SKSE/Logger.cpp
+++ b/src/SKSE/Logger.cpp
@@ -73,7 +73,7 @@ namespace SKSE
 
 			std::filesystem::path path = knownPath.get();
 			path /= "My Games"sv;
-			path /= *REL::Relocation<const char**>(REL::ID(380738)).get();
+			path /= *REL::Relocation<const char**>(REL::ID(502114)).get();
 			path /= "SKSE"sv;
 			return path;
 		}
